setup.py

```diff
@@ -14,8 +14,15 @@
     include_package_data=True,
     packages=find_packages(),
     install_requires=[
-        'django>=1.6',
-        'gitpython>=0.3.2.RC1',
+        'django>=1.11',
+        'gitpython>=2.1.12',
+        'click==7.0',
+        'requests>=2.22.0',
+    ],
+    scripts=[
+        'project_info/bin/release-version.py',
+        'project_info/bin/release-git.py',
+        'project_info/bin/release-bitbucket.py',
```

Comment:
@radimsuckr your command for ecs should have separate file

Reply:
Yeah, I've already separated it. :+1: